content/projects/gsoc/students.adoc

```diff
@@ -87,63 +292,23 @@ between students, org admins and mentors.
-. Attend Jenkins governance meetings if the timezone allows
-. Spend a significant amount of time on study and design during the community bonding
-
-=== Coding period
-
-Students are expected to...
-
-. Work on the GSoC project as it is a full-time employment.
- * It means that 30..40 hours per week is an **expected** workload though it can be adjusted upon the agreements with mentors.
- * It also means that you have ~5 "vacation days" during the project, do not hesitate to use them if required
-. Use weekend to have a rest, avoid significant overwork and enjoy coding
-. Timely notify mentors in the case of emergencies and outages (missing scheduled meetings, etc.).
-. Timely notify mentors and org admins about unexpected time commitments
-. Be around in _#jenkins_ IRC and in the project chats during the working hours
-. Attend Jenkins governance meetings if the timezone allows
-. Be proactive; reach out to the community if required
-. Produce the good quality code with reasonable amount of testing and documentation
-. Have a finalized deliverable at the end of the project
-
-Students are **not** expected to...
-
-. Strictly follow the originally submitted mini-design and the project proposal
- * The world is not ideal, and there may be unexpected obstacles or shortcuts
- * Upon the discussion with mentors, any plan can be adjusted
- * We expect students to achieve at least some goals in the original proposal
-. Investigate and solve *every* issue on their own
- * We have mentors and experts, who can help you by answering questions and doing joint investigation if required
-
-=== Evaluations
+Once GSoC is over, final results are announced by Google. But this is not the end of the road.
 
-. At the end of the each coding phase students and mentors present the project status at the public meeting
-. As a part of the Final evaluation students present the project results at the link:https://www.meetup.com/Jenkins-online-meetup/[Jenkins Online Meetup]
+You can:
 
-=== Post-GSoC
+. Continue to develop your project within the Jenkins community
+. Participate in other Jenkins projects
+. Participate again next year
+. Become a mentor in link:https://summerofcode.withgoogle.com[Google Summer of Code] for next year
+. Become a mentor in link:https://codein.withgoogle.com/[Google Code In]
 
 Depending on the project results, and available budget, we may offer a sponsored trip
-to link:https://www.cloudbees.com/jenkinsworld/home[Jenkins world] or another Jenkins-related event to students
+to link:https://www.cloudbees.com/devops-world/[DevOps World - Jenkins World] or another Jenkins-related event to students
 who successfully finish their projects.
 This sponsorship is not guaranteed though.
 
-If students agree to go to such event, we expect them to present their project and to write a blog-post about the trip.
+If students agree to go to such event, we expect students to present your project and to write a blog-post about the trip.
```

Comment:
Add link to the blogpost from @pvtuan10 ?

Reply:
got it